build_tools/docker/cmake_android/gcr_update.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Builds and pushes the cmake image to gcr.io/iree-oss/
+
+set -x
+set -e
+
+# Ensure correct authorization.
+gcloud auth configure-docker
+
+# Determine which image tag to update. Updates :latest by default.
+TAG="latest"
+if [[ "$#" -ne 0  ]]; then
+  if [[ "$@" == "--update-prod" ]]; then
+    TAG="prod"
+  else
+    echo "Invalid commandline arguments. Accepts --update-prod or no arguments."
+    exit 1
+  fi
+fi
+echo "Updating ${TAG}"
+
+
+# Build and push the cmake image.
+docker build --tag "gcr.io/iree-oss/cmake-android:${TAG}" build_tools/docker/cmake/
+docker push "gcr.io/iree-oss/cmake-android:${TAG}"
```

Comment:
So we should probably refactor so we only need one script. This should just be:

`update_gcr cmake_android`

(not this PR). @phoenix-meadowlark could you take a look at that?

Reply:
Something like
```shell
./build_tools/docker/update_gcr.sh --image=bazel --update-prod
```
Where `image ∈ {bazel, bazel_bindings, bazel_tensorflow, cmake, cmake_android, rbe}` maps to a bunch of conditional statements? (Switch statements with drop through would work if we had chain dependencies, but we have `bazel -> bazel-bindings` and `bazel -> bazel-tensorflow`).

Reply:
Yes. Except per the organization point below, I think the images should be called `linux/bazel/core`, `linux/bazel/bindings`, `android/cmake`, etc.

Check out `getopt` for shell command line flag parsing.

Reply:
We should probably also rename all of our images then. E.g. `gcr.io/iree-oss/linux/bazel/core` and so on :)